### ISSUE TITLE
[FEATURE] Adding a news or updates section, that will give the user daily updates on what's happening in the world.

### ISSUE THREAD

**Issue by sanchitaa14** [narayan954/dummygram#487]
### Description

Just seeing posts from people would not make the site more attractive, but if there was another section called updates, like a for you page but rather in that people actually get updates on what's happening in the world or near them, the hot topics that are occuring right now.

### Screenshots

![image](https://github.com/narayan954/dummygram/assets/118709611/f4a79184-7b0d-463f-a119-aa9077899da6)
something like this but with better css, which i can take care of as per what you would prefer.

**Comment by sanchitaa14** [comment]
Yes, I would very much like to work on this issue

**Comment by narayan954** [comment]
> ### Description
> Just seeing posts from people would not make the site more attractive, but if there was another section called updates, like a for you page but rather in that people actually get updates on what's happening in the world or near them, the hot topics that are occuring right now.
> 
> ### Screenshots
> ![image](https://user-images.githubusercontent.com/118709611/246494054-f4a79184-7b0d-463f-a119-aa9077899da6.png) something like this but with better css, which i can take care of as per what you would prefer.
> 
> ### Additional information
> _No response_

Hi @sanchitaa14 this seems like a great idea, but where'd we implement it? Are you suggesting something like a separate section called "Explore" in the sidebar that renders basically a mini news application, maybe playing shorts/news article arranged as clickable tiles in n*m grid, playable as a popup on click ???

**Comment by sanchitaa14** [comment]
yes, a separate section for this, and i could also try adding extra features of popup if you want

**Comment by narayan954** [comment]
> yes, a separate section for this, and i could also try adding extra features of popup if you want

Yes, you can try. Is level 2 difficulty okay for now? level 3 label if you're able to implement the popup feature!

**Comment by sanchitaa14** [comment]
ok got it

**Comment by narayan954** [comment]
> ok got it

great, best of luck!

**Comment by Palakkgoyal** [comment]
Hey @narayan954 Can I work on this?

**Comment by narayan954** [comment]
> Hey @narayan954 Can I work on this?

Sure, you can
